src/sentry/issues/escalating.py

```diff
@@ -315,8 +315,6 @@ def manage_issue_states(
         if updated:
             add_group_to_inbox(group, GroupInboxReason.ESCALATING, snooze_details)
             record_group_history(group, GroupHistoryStatus.ESCALATING)
-            group.status = GroupStatus.UNRESOLVED
-            group.substatus = GroupSubStatus.ESCALATING
```

Comment:
Any reason why you're removing these assignments?

Reply:
Assigning the values shouldn't be updating the actual db. I think we still want to set the values on the model since the group might be used further downstream.

Reply:
`update()` only returns the number of rows that matches the criteria, so it won't return the entire models back. It's not ideal, but I think just assigning the value to the passed in group is how we wanna do this.